src/app/app.component.html

```diff
@@ -112,10 +77,10 @@
         <th style="color: green"> Matched</th>
       </tr>
       <tr>
-        <th>Criminal Record Status from CCTNS/NCRB/ICJS</th>
+        <th>Criminal Record Status (Alert Level)</th>
```

Comment:
Don't delete Record Status from CCTNS/NCRB/ICJS. Add a new line for Record STatus (Alert Level )